packages/transport/src/types/messages.ts

```diff
@@ -1459,8 +1468,9 @@ export enum Enum_SafetyCheckLevel {
 export type SafetyCheckLevel = keyof typeof Enum_SafetyCheckLevel;
 
 export enum Enum_HomescreenFormat {
-    Toif144x144 = 1,
-    Jpeg240x240 = 2,
```

Comment:
was this ever released to production? do we need to care about backwards compatibility?

Reply:
Current `2.6.0`
![Screenshot 2023-07-03 at 11 09 55](https://github.com/trezor/trezor-suite/assets/33235762/4509869d-71c9-41d5-bbb0-749bc0e97b3a)

So we need to care about backwards compatibility as we use it in `isHomescreenSupportedOnDevice` method to check if homescreen feature is available on device

Reply:
protobuf doesn't care about names. you can use the new definition on 2.6.0 and will get Jpeg insted of Jpeg240x240, so the only thing you need to handle is filling in the right dimensions if the firmware doesn't send them